cms/auth/middleware.py

```diff
@@ -15,6 +15,7 @@
 logger = logging.getLogger(__name__)
 
 JWT_SESSION_ID_KEY = "jwt_session_id"
+BYPASSED_INTERNAL_PATHS = {"/-/readiness", "/-/liveness", "/-/health", "/health"}
```

Comment:
Suggestion: Exclude everything starting with `/-/`, as I've intentionally used that namespace for these internal URLs. That way, there's no need to keep updating this set.